packages/vue/src/components/force-new-password.vue

```diff
@@ -56,15 +57,8 @@ const onForceNewPasswordSubmit = (e: Event): void => {
 };
 
 const submit = (e: Event): void => {
-  const formData = new FormData(<HTMLFormElement>e.target);
-  send({
-    type: 'SUBMIT',
-    //@ts-ignore
-    data: {
-      //@ts-ignore
-      ...Object.fromEntries(formData),
-    },
-  });
+  e.preventDefault();
```

Comment:
+1 you get the idea :)

Reply:
Ah, thanks for that -- fixed by https://github.com/aws-amplify/amplify-ui/pull/1366/commits/269031200f781ff0e32b8f736389a05fb616bb44!